quickannotator/api/v1/annotation/helper.py

```diff
@@ -83,11 +83,11 @@ def count_annotations_within_bbox(table, x1, y1, x2, y2):
     result = qadb.db.session.execute(stmt).scalar()
     return result
 
-def retrieve_annotation_table(image_id: int, annotation_class_id: int, is_gt: bool) -> Table:
+def retrieve_annotation_table(session, image_id: int, annotation_class_id: int, is_gt: bool) -> Table:
```

Comment:
i can't see everything from this scope in the code review - but is there a reason to pass in session here as a parameter versus creating it dynamically? is this function being called within the scope of some type of larger tasks which might need rollback?